.github/workflows/codeql.yml

```diff
@@ -0,0 +1,57 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "7.0", "3.13", "3.14", "3.15", "3.16" ]
+  pull_request:
+    branches: [ "7.0" ]
+  schedule:
+    - cron: "2 10 * * 0"
```

Comment:
Schedule would also run it at 10:02 AM every Sunday, not sure if that's UTC though.
Do we want this?

Reply:
It might be best to remove the `schedule:`.

Last I checked, github will automatically disable workflows with scheduling in some circumstances.  eg. no activity in 2 months.  While disabled, all events are ignored, including pushes and pull requests.  I found this to be ... surprising.